### PR TITLE
Typos and shortening

### PR DESCRIPTION
**Pull request by tristaaan**
Shortening includes dropping some sentences and going some \maketitle margin hacks.

### CODE REVIEW

File: report/main.tex [hunk 1]
```diff
@@ -69,9 +73,9 @@
 \input{sections/secondary-groups.tex}
 \input{sections/conclusion.tex}
 
-\begin{appendix}
+\begin{appendices}
```
Comment (tristaaan) [inline report/main.tex]: I didn't realize this change was here, looks like it makes no difference though

File: report/sections/results.tex [hunk 2]
```diff
@@ -31,7 +31,7 @@ \section{Results}
...
+With \knn methods we can attain an average accuracy of $46.5\%$ with a standard deviation of $0.045$; this is a mediocre score. This difference in accuracy compared to other methods can be thought of as a problem with clustering and the nature of the landmark points. Any set of emotions where, for example, an eyebrow point is more or less the same point (e.g. surprised and happy), will be classified by kNN as the same emotion, no other points will be taken into account. Among our labels are two super-categories of emotions, positive (happy, surprised) and negative (fear, angry, disgust, sad). The reader is invited to make those emotions with his or her face and take note the placement of their eyebrows. They may notice they are in the same place for each emotion, other features may be in the same place too.
```
Comment (tristaaan) [inline report/sections/results.tex:34]: semi-colon after `$0.045$`

File: report/sections/secondary-groups.tex [hunk 3]
```diff
@@ -5,14 +5,14 @@ \section{Secondary Groups}
 
 \subsection{Computer Vision Group}
 %Format read from CV Group
-After training our models with the Bosphorus Database we need to predict an emotion by reading input from the CV group. They are creating an array containing $25$ landmarks and passing it as input into our models. PCA was used for dimensionality reduction before training our model, so the same PCA model is used on the given input for reducing the number of features.
```
Comment (tristaaan) [inline report/sections/secondary-groups.tex:8]: I felt like mentioning the Bosphorus Database over and over was redundant, they already know what dataset we're using

**Comment by ankurshukla03** [comment]
Should we have references before appendix in the report?

**Comment by diegocasmo** [comment]
> Should we have references before appendix in the report?

Yes. [See more](http://answers.library.westernsydney.edu.au/faq/40051), [or here](http://libraryconnect.libanswers.com/faq/111958), [etc](http://writeanswers.royalroads.ca/faq/199193)

**Comment by tristaaan** [comment]
I'm finding that too, although I'm never clear about what document styling
we're supposed to adhere to here. (APA = American Psychological
Association).

Let's reorder it.

-Tristan Wright

**Comment by ankurshukla03** [comment]
Here's the link of the [report.](https://www.sharelatex.com/read/dyvgnjbssbmk)

**Review by ankurshukla03 — APPROVED** [review]

**Review by diegocasmo — CHANGES_REQUESTED** [review]
Still > 5 pages

**Comment by diegocasmo** [comment]
> Here's the link of the [report.](https://www.sharelatex.com/read/dyvgnjbssbmk)

A few references are undefined in that link (see last paragraph)

**Comment by ankurshukla03** [comment]
> @misc{muppets,
author={Jim Henson, Disney},
title={The Muppets},
howpublished={\url{https://muppets.disney.com/}},
notes={Online; Accessed May 31 2018}
}

Tristan added one more references in bib file, that was the problem now it's fixed. Maybe we can shorten the KNN section in results?

**Comment by tristaaan** [comment]
Something is not getting carried over? This fits in 5 pages on mine....

**Review by diegocasmo — APPROVED** [review]

**Pull request merged**